smart_card_connector_app/src/testing-smart-card-simulation-libusb-hook.js

```diff
@@ -110,7 +110,7 @@ GSC.TestingLibusbSmartCardSimulationHook = class extends GSC.LibusbProxyHook {
         remoteCallMessage.makeRequestPayload());
     // Hack to prevent spurious "UnhandledPromiseRejection" errors, which happen
     // when using the native `await` against Closure Library's (now-deprecated)
-    // `goog.Promise` if the latter is rejected.
+    // `goog.Thenable` if the latter is rejected.
```

Comment:
I think this should still say goog.Promise

Reply:
done, thanks for catching this!